### PR TITLE
Fix rust cargo audit (Uplift to 1.35.x)

### PR DESCRIPTION
**Pull request by bsclifton**
Manual uplift of https://github.com/brave/brave-core/pull/11890

Fixes cargo advisories brought in by the SKU portion of https://github.com/brave/brave-core/pull/11985

NO TICKET

### CODE REVIEW

**Comment by bsclifton** [comment]
Pushing a fix for lint (raised in https://ci.brave.com/blue/organizations/jenkins/pr-brave-browser-bsc-fix-audit-errors-1.35.x-noplatform/detail/pr-brave-browser-bsc-fix-audit-errors-1.35.x-noplatform/1/pipeline). Will mark as `CI/Skip` to avoid another run

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.35.x` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
